scripts/train_pythae_vae.py

```diff
@@ -136,7 +141,7 @@ def _get_model_classes(dataset: str, network_type: Optional[str] = None) -> Tupl
         The encoder and decoder class.
     """
     if dataset == "dsprites":
-        return None, None  # Not implemented yet
+        return lambda c: conv_models.ConvEncoder(c, n_channels=1), lambda c: conv_models.ConvDecoder(c, n_channels=1)
```

Comment:
Somewhat hacky, but I see the reason :slightly_smiling_face:

Reply:
Yes :D